add explanation for clearing shared segments

garyverhaegen-da opened the pull request:
As requested on #6530.

SamirTalwar-DA commented on ci/clear-shared-segments-macos.yml:
```diff
@@ -0,0 +1,32 @@
+steps:
+  - bash: |
+      set -euo pipefail
+      for shmid in $(ipcs -m | sed 1,3d | awk '{print $2}' | sed '$d'); do ipcrm -m $shmid; done
```
I'd find this easier to read if the important operation was on a new line.

```suggestion
      for shmid in $(ipcs -m | sed 1,3d | awk '{print $2}' | sed '$d'); do
        ipcrm -m $shmid
      done
```

SamirTalwar-DA commented:
Also what @cocreature said. (He's too fast.)

SamirTalwar-DA commented:
FWIW, I still think this is a good improvement.

garyverhaegen-da commented:
> LGTM but I understood @SamirTalwar’s suggestion as turning this into a shell script so you can easily run it locally not a yaml file.

Ah, this was actually addressing your comment. I hadn't seen @SamirTalwar-DA's 😓. Will get on that then.
